src/NuGet.Services.AzureSearch/Models/SearchDocument.cs

```diff
@@ -43,8 +43,12 @@ public class UpdateLatest : BaseMetadataDocument, IVersions, IOwners
             [IsFilterable]
             public string SearchFilters { get; set; }
 
+            [IsFilterable]
+            public string[] PackageTypes { get; set; }
```

Comment:
can we put [Analyzer(ExactMatchCustomAnalyzer.Name)] on here too?

Reply:
nit: Personally I would switch the names to `FilterablePackageTypes` and `PackageTypes` -- I would say in general the `PackageTypes` are the real values and the filterable ones are a special case

Reply:
Yea we can.  I wasn't sure if that was meaningful on filterable, since I think this requires exact match? I can definitely add it.

Reply:
swapped name specificity and added the exact match analyser attribute.